src/Video/ParticleSystemRenderer.hpp

```diff
@@ -0,0 +1,134 @@
+#pragma once
+#include <glm\glm.hpp>
+#include "Shader\Shader.hpp"
+#include "Shader\ShaderProgram.hpp"
+
+#include "linking.hpp"
+
+struct ParticlePos {
+    float x;
+    float y;
+    float z;
+    float w;
+};
+
+struct ParticleVelocity {
+    float vx;
+    float vy;
+    float vz;
+    float life;
+};
+
+struct ParticleColor {
+    float cx;
+    float cy;
+    float cz;
+    float ca;
+};
+
+struct ParticleModelMatrix {
+    glm::mat4 m = glm::mat4(1);
+};
+
```

Comment:
These structs should be put in a namespace.

Reply:
Bump.